functions.lua

```diff
@@ -80,6 +80,12 @@ function travelnet.is_elevator(node_name)
 	return node_name == "travelnet:elevator"
 end
 
+function travelnet.is_travelnet_or_elevator(pos)
+	local node = minetest.get_node(pos)
+	local node_def = minetest.registered_nodes[node.name]
+	return node_def.groups.travelnet or node_def.groups.elevator
```

Comment:
is it guaranteed that every possible node has groups table, also ignore nodes?

if not or not sure then maybe
```suggestion
	return node_def and node_def.groups and (node_def.groups.travelnet or node_def.groups.elevator)
```
node_def should be checked anyway (for unknown nodes) so at least
```suggestion
	return node_def and (node_def.groups.travelnet or node_def.groups.elevator)
```

Reply:
Oh yeah I just checked, if you register it without groups it won't have any, best to check both